CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   - Highlight named lambda functions properly.
   - Fix syntax highlighting for functions and vars with metadata on the previous
     line.
+- Improve semantic indentation rules to be more consistent with cljfmt.
```

Comment:
I'd also mention the new defcustom in the changelog.

Reply:
Done.